### PR TITLE
add unity-example

### PR DESCRIPTION
**Pull request by vegabs**

### CODE REVIEW

File: examples/endpoint_unity_example/readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Octupus Sensing - Unity Example
+In this example, We're going to send predefined messages (START, STOP and, TERMINATE) from Unity to Octopus sensing, to control the recording of data. Octopus sensing (Python) is used as a server and Unity (C#) as a client.
+
```
Comment (nastaran62) [inline examples/endpoint_unity_example/readme.md:3]: You are not using Octopus Sensing as the server. Octopus Sensing is a library. You have a server in python developed using the Octopus Sensing library.

File: examples/endpoint_unity_example/readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Each button has the ability to send a specific message when pressed. See the UI below.
+
+![](ui-unity.jpg)
```
Comment (nastaran62) [inline examples/endpoint_unity_example/readme.md:6]: Which image?

File: examples/endpoint_unity_example/requestController.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        form.AddField("stimuli_id", stimuli_id);
+
+        string jsonData = "{\"type\":\"" + type + "\",\"experiment_id\":\"" + experiment_id + "\",\"stimulus_id\":\"" + stimuli_id + "\"}";
+
+        if (type == "TERMINATE")
+        {
+           jsonData = "{\"type\":\"TERMINATE\"}";
+        }
+
+        UnityWebRequest www = new UnityWebRequest("http://127.0.0.1:9331", "POST");
```
Comment (nastaran62) [inline examples/endpoint_unity_example/requestController.cs:62]: Please add comments for each of these lines

File: examples/endpoint_unity_example/requestController.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    public Button startStimuli;
+    public Button stopStimuli;
+    public Button terminateExperiment;
+
+    string stimuli_id = "10";
+    string experiment_id = "00";
+
+    void Start()
+    {
+        startStimuli.onClick.AddListener(startRequest);
```
Comment (nastaran62) [inline examples/endpoint_unity_example/requestController.cs:19]: And add comments for this

**Review by nastaran62 — CHANGES_REQUESTED** [review]

**Comment by vegabs** [comment]
I've added comments and the image

**Pull request merged**